lib/fabrik/DirectorManager.js

```diff
@@ -44,7 +43,7 @@ class DirectorManager extends BaseManager {
     return _(this.settings)
       .chain()
       .get('stemcell', {})
-      .defaults(infrastructure.stemcell)
+      .defaults(this.director.config.infrastructure.stemcell)
```

Comment:
comments similar to [above](https://github.com/SAP/service-fabrik-broker/pull/7/files#r133368904)